readyapi4j-core/src/main/java/com/smartbear/readyapi4j/execution/RecipeExecutor.java

```diff
@@ -12,13 +11,54 @@
 
 public interface RecipeExecutor {
 
+    /**
+     * Submit a recipe for asynchronous execution.
+     *
+     * @param jsonText recipe to be executed.
+     * @return an instance of <code>Execution</code> containing the current state of execution
+     */
+    Execution submitRecipe(String jsonText);
+
+    /**
+     * Submit a recipe for synchronous execution.
+     *
+     * @param jsonText recipe to be executed.
+     * @return an instance of <code>Execution</code> containing the execution result
+     */
+    Execution executeRecipe(String jsonText);
```

Comment:
Ok, I will remove the methods from API, how about adding it as a static method in the TestRecipe or TestRecipeBuilder class rather than creating a new class TestCaseSupport?

public static TestRecipe createFrom(String jsonText)